### PR TITLE
add describe about WordPress deploy yml

### PR DESCRIPTION
**Pull request by Coderhypo**
fix https://github.com/kubernetes/website/issues/14953

I reference the wordpress-deployment.yaml and add describe about it.

### CODE REVIEW

File: content/en/docs/tutorials/stateful-application/mysql-wordpress-persistent-volume.md [hunk 1]
```diff
@@ -93,6 +93,10 @@ The following manifest describes a single-instance MySQL Deployment. The MySQL c
 
 {{< codenew file="application/wordpress/mysql-deployment.yaml" >}}
 
+The following manifest describes a single-instance WordPress Deployment. The wordpress container mounts the PersistentVolume at /var/www/html for website data file. The `WORDPRESS_DB_HOST` environment variable sets the Service name of MySQL defined above, and wordpress will access the database by Service. The `WORDPRESS_DB_PASSWORD` environment variable sets the database password from the Secret kustomize generated.
```
Comment (rajeshdeshpande02) [inline content/en/docs/tutorials/stateful-application/mysql-wordpress-persistent-volume.md]: The following manifest describes a single-instance WordPress Deployment. The WordPress container mounts the PersistentVolume at /var/www/html for a website data file. The WORDPRESS_DB_HOST environment variable sets the Service name of MySQL defined above, and WordPress will access the database by Service. The WORDPRESS_DB_PASSWORD environment variable sets the database password from the Secret kustomize generated.

Comment (rajeshdeshpande02) [reply content/en/docs/tutorials/stateful-application/mysql-wordpress-persistent-volume.md]: Please make these minor corrections

**Comment by Coderhypo** [comment]
@rajeshdeshpande02 Thanks for your review, and I make some corrections

**Pull request merged**